solidity/contracts/token/TokenBridgeCctp.sol

```diff
@@ -137,8 +140,28 @@ contract TokenBridgeCctp is HypERC20Collateral, AbstractCcipReadIsm {
             (bytes, bytes)
         );
 
+        bytes calldata tokenMessage = _hyperlaneMessage.body();
+        require(
+            tokenMessage.length == CCTP_TOKEN_BRIDGE_MESSAGE_LEN,
```

Comment:
I think we can add one internal  `_checkLen(tokenMessage)` function that reverts if we don't have the right length. This would replace the assert on line 214.

Reply:
one is calldata and one is memory so unfortunately solidity doesnt let us share logic easily unless we memcopy